address_book_ui/src/app/test/test.component.ts

```diff
@@ -1,9 +1,10 @@
 import { Component } from '@angular/core';
+import { SidePannelComponent } from '@app/core/side-pannel/side-pannel.component';
 
 @Component({
```

Comment:
It might be a good idea to wrap the test and record components into a "components" folder, similar to how we organize the "pages" folder.